src/package-linker.js

```diff
@@ -304,8 +304,15 @@ export default class PackageLinker {
       const stat = await fs.lstat(entryPath);
 
       if (stat.isSymbolicLink()) {
-        const packageName = entry;
-        linkTargets.set(packageName, await fs.readlink(entryPath));
+        try {
```

Comment:
eslint errors on this.
Please run `yarn run prettier` to autoformat the code

Reply:
Will do, and will check with the linter before committing. Thanks.

Reply:
Done. yarn lint completed.